src/babel/index.ts

```diff
@@ -17,6 +23,7 @@ export type BabelPluginOptions = Partial<{
   client: boolean;
   modulePathReplacer: ModulePathReplacer;
   noopRedundantLoaders: boolean;
+  resolverOptions: Partial<ResolveOptions> | undefined;
```

Comment:
Can you document the `resolverOptions` in `docs/tooling/babel-plugin`? It's pretty manual right now, but something like this below the `noopRedundantLoaders` section should be sufficient:

> ---
>
> ### `resolverOptions?`
> `ResolverOptions = { extensions: ['.tsx', '.ts', '.jsx', '.js'] }`
> 
> Overrides the options passed to [enhanced-resolve](https://github.com/webpack/enhanced-resolve), which is used to generate the [`moduleId`](api/lazy?id=moduleid) by resolving the path to the [loader](api/lazy?id=loader).
> 
> This option should be configured to match any custom file resolution that occurs in your application. For example, in webpack the options defined in [`resolve`](https://webpack.js.org/configuration/resolve) should be passed into `resolverOptions`.

**Source:**
```
---

### `resolverOptions?`
`ResolverOptions = { extensions: ['.tsx', '.ts', '.jsx', '.js'] }`

Overrides the options passed to [enhanced-resolve](https://github.com/webpack/enhanced-resolve), which is used to generate the [`moduleId`](api/lazy?id=moduleid) by resolving the path to the [loader](api/lazy?id=loader).

This option should be configured to match any custom file resolution that occurs in your application. For example, in webpack the options defined in [`resolve`](https://webpack.js.org/configuration/resolve) should be passed into `resolverOptions`.
```

Reply:
@MonicaOlejniczak Changed!